managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -249,6 +250,10 @@ public ManagedLedgerImpl(ManagedLedgerFactoryImpl factory, BookKeeper bookKeeper
         this.config = config;
         this.store = store;
         this.name = name;
+        this.ledgerMetadata = ImmutableMap.of(
+                "application", "pulsar".getBytes(StandardCharsets.UTF_8),
```

Comment:
`"pulsar".getBytes(StandardCharsets.UTF_8)` Could be stored in a static var so that we only get the bytes once.